src/common/mfu_daos.c

```diff
@@ -143,7 +148,7 @@ static int daos_parse_path(
         } else {
             strcpy(path, dattr.da_rel_path);
         }
-    } else if (strncmp(path, "daos://", 7) == 0) {
+    } else if (strncmp(path, "daos:", 5) == 0) {
```

Comment:
@daltonbohning why don't we check the whole daos prefix here? Just wondering really since it looks like you had it before.

Reply:
@dsikich I was trying to handle the case where maybe the user just type the format incorrectly.
For example:
`daos:/pool/cont`
In this case, it's safe to say that the "daos:" prefix meant that the user wants this to be DAOS, so we can error and alert them.

It's probably incredibly unlikely that an _**actual**_ file/dir would start with "daos:".

What do you think?

Reply:
@daltonbohning yeah probably unlikely especially with the check for the colon added. In that case it might help to print out a message to the user reminding them of the correct format.

Reply:
@daltonbohning I can see people making this mistake or missing the second forward slash.

Reply:
@dsikich I added a more descriptive message.